.storybook/vue/create-vue-bindings-from-props.ts

```diff
@@ -7,7 +7,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import pickBy from 'lodash.pickby';
+import pickBy from 'lodash-es/pickby';
```

Comment:
Forgot to mention this thing, my apologies 🙇 

```suggestion
import pickBy from 'lodash-es/pickBy';
```

Reply:
My fault! haha Updated!